.github/workflows/test-enos-scenario-ui.yml

```diff
@@ -135,7 +135,6 @@ jobs:
         id: run_ui_tests
```

Comment:
This step doesn't run anything, so should be removed ?

Reply:
Fixed.